src/LeftNav/LeftNav.less

```diff
@@ -0,0 +1,80 @@
+@import (reference) "../less/index.less";
+
+.LeftNav {
+  .shade(color, @primary_blue, 2);
+  background: @neutral_off_white;
+  user-select: none;
+  .text--light;
+  // Maintain computed width based on content
+  // (i.e. don't let the NavLink labels wrap)
+  .flexShrink(0);
+}
+
+.LeftNav--topnav {
+  .list-reset;
+  min-width: 9rem;
+  box-shadow: @borderWidthM 0 @borderWidthM @neutral_silver;
+  .zIndex--1; // Ensure the shadow goes over the subnav
+}
+
+.LeftNav--subnav {
+  .list-reset;
+  .tint(background, @primary_blue, 6);
+  color: @neutral_black;
+  min-width: 9rem;
+  box-shadow: @borderWidthM 0 @borderWidthM @neutral_silver;
+}
+
+.NavLink {
+  .padding--left--m;
+  .padding--right--xl;
```

Comment:
but then you have to remember the order of the four values. i find that a lot harder to read, so i want to stick with this